hooks/collector.py

```diff
@@ -241,6 +241,14 @@ def _collect_file(self, parent_item, path, frame_sequence=False):
 
             # disable thumbnail creation since we get it for free
             file_item.thumbnail_enabled = False
+        else:
+            # Try to generate a thumbnail from the file
+            try:
+                thumbnail = publisher.util.get_thumbnail(path, file_item.context)
+                if thumbnail:
+                    file_item.thumbnail = thumbnail
```

Comment:
can `thumbnail` be `None`? In this case, this can raise to it can go the except block and do the log.

Reply:
After reading the util file changes, this error handling can be relocated to the `get_thumbail` method.

Reply:
It is not an error if the thumbnail is None, it just means that the file is not supported for automatic thumbnail generation. If there was an error to report, it is caught in the TankError and Exception except blocks.

I've updated this to set the file item thumbnail regardless, the file item thumbnail can be None (and is None by default)